worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/EllipsoidRegionSelector.java

```diff
@@ -93,6 +93,7 @@ public EllipsoidRegionSelector(RegionSelector oldSelector) {
             BlockVector3 center = pos1.add(pos2).divide(2).floor();
             region.setCenter(center);
             region.setRadius(pos2.subtract(center).toVector3());
+            started = true;
```

Comment:
Should this flag also be inherited in the other branch of the `if`?

Reply:
Ah yes - applied there.